plugins/node/opentelemetry-instrumentation-express/examples/package.json

```diff
@@ -35,7 +35,7 @@
     "@opentelemetry/instrumentation": "^0.27.0",
     "@opentelemetry/instrumentation-http": "^0.27.0",
     "@opentelemetry/resources": "^0.26.0",
-    "@opentelemetry/sdk-trace-base": "^0.26.0",
+    "@opentelemetry/sdk-trace-base": "^1.0.0",
     "@opentelemetry/sdk-trace-node": "^0.26.0",
```

Comment:
I still think we should avoid 0.x and 1.x versions of core components.
But as far as I understood other comments the target is to update dependencies in followup PRs.

Reply:
Good catch. I went ahead and bumped all of the otel dependency versions to their latest.